project-fortis-interfaces/src/components/Insights/Dashboard.js

```diff
@@ -246,6 +254,14 @@ export default class Dashboard extends React.Component {
               fetchCsvs={this.refreshDashboardWithCsv}
             />
           </div>
+          <div className="dashboard-action">
+            <MapBoundingReset
+              tooltipOn="Click to reset map boundaries"
```

Comment:
These two tooltip properties are not used in the component. You can use the [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) library to document the props that your component requires in order to get tooling support for highlighting unused properties (however, PropTypes unfortunately isn't used much in Fortis currently so don't feel like this is a requirement for merging, this is more just an informational comment).

Reply:
I think these changes actually didn't get pushed up, i removed them. Pushing up more changes now